Makefile

```diff
@@ -1,6 +1,8 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+OP_IMG ?= leiwanjun/fluentbit-operator:latest
```

Comment:
shouldn't be your own repo?
why not using the original controller:latest any more?

Reply:
migtator is to migrate fluentbit operator v0.1.0 crd to v0.2.0, maybe we needn't to put it into Makefile